packages/now-client/src/deploy.ts

```diff
@@ -102,6 +105,18 @@ export default async function* deploy(
         : getDefaultName(options.path, options.isDirectory, files)
   }
 
+  if (options.totalFiles === 1 && !metadata.builds && !metadata.routes) {
+    const filePath = Array.from(files.values())[0].names[0]
+    const segments = filePath.split('/')
+
+    metadata.routes = [
+      {
+        src: '/',
+        dest: `/${segments[segments.length - 1]}`
+      }
+    ]
+  }
```

Comment:
What is the purpose of this new behavior? Can you add a test?

Reply:
This technically isn't new, it fixes a bug. `now-client` was deploying a single file as a file listing, this behavior was missing which was a bug